web/src/tippyjs.js

```diff
@@ -260,6 +259,32 @@ export function initialize() {
         appendTo: () => document.body,
     });
 
+    delegate("body", {
+        target: ".recipient_bar_icon",
+        onShow(instance) {
+            if (!document.body.contains(instance.reference)) {
+                return false;
+            }
+            const $elem = $(instance.reference);
+
+            const config = {attributes: false, childList: true, subtree: true};
+            const target = $elem.parents(".message_header.message_header_stream.right_part").get(0);
+            const nodes_to_check_for_removal = [
+                $elem.parents(".recipient_bar_controls").get(0),
+                $elem.get(0),
+            ];
+            hide_tooltip_if_reference_removed(target, config, instance, nodes_to_check_for_removal);
+            return true;
+        },
+        onHidden(instance) {
+            instance.destroy();
+            if (observer) {
+                observer.disconnect();
+            }
+        },
+        appendTo: () => document.body,
+    });
```

Comment:
It would be nice if we can compress this to involve less duplicated code somehow.

Reply:
We can leave that for a follow-up, but I think we should make this a function that can be called, taking just the selectors it needs as parameters.

Reply:
Sounds good. Since we need to apply similar changes to multiple tooltips and having a function can significantly reduce code duplication, I will open a follow-up PR to implement a dedicated function for this purpose.